.circleci/config.yml

```diff
@@ -21,7 +21,8 @@ jobs:
             - yarn-packages-v1-{{ .Branch }}-{{ checksum "yarn.lock" }}
             - yarn-packages-v1-{{ .Branch }}-
             - yarn-packages-v1-
-      - run: 'npx greenkeeper-lockfile@next greenkeeper-lockfile-upload'
+      - run: npx greenkeeper-lockfile@next greenkeeper-lockfile-upload
+      - run: yarn install --no-progress --silent
```

Comment:
Not sure how we got away with not having this. Maybe I'm missing something 🤷‍♂

Reply:
Oh. Maybe it was the caching of the modules? Good catch.